test(jans-auth-server): java.lang.NoClassDefFoundError: com/couchbase/client/java/env/ClusterEnvironment

### What is failing?
Different server side tests are failing with NoClassDefFoundError.
It still have references to couchbase however it CB and spanner support was removed from AS.

java.lang.NoClassDefFoundError: com/couchbase/client/java/env/ClusterEnvironment
https://jenkins.jans.io/jenkins/job/jans-auth-server/15178/console

```
09:08:08,699 [ERROR] io.jans.as.server.comp.db.UserServiceTest.testUserAddAuthenticator -- Time elapsed: 0.025 s <<< ERROR!
java.lang.NoClassDefFoundError: com/couchbase/client/java/env/ClusterEnvironment
	at java.base/java.lang.Class.getDeclaredFields0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredFields(Class.java:3297)
	at java.base/java.lang.Class.getDeclaredFields(Class.java:2371)
	at org.junit.platform.commons.util.ReflectionUtils.getDeclaredFields(ReflectionUtils.java:1496)
	at org.junit.platform.commons.util.ReflectionUtils.findAllFieldsInHierarchy(ReflectionUtils.java:1225)
	at org.junit.platform.commons.util.ReflectionUtils.findFields(ReflectionUtils.java:1213)
	at org.junit.platform.commons.util.AnnotationUtils.findAnnotatedFields(AnnotationUtils.java:491)
	at org.junit.platform.commons.util.AnnotationUtils.findAnnotatedFields(AnnotationUtils.java:468)
	at org.junit.platform.commons.support.AnnotationSupport.findAnnotatedFields(AnnotationSupport.java:288)
	at org.jboss.weld.junit5.auto.ClassScanning.scanForRequiredBeanClasses(ClassScanning.java:90)
	at org.jboss.weld.junit5.auto.WeldJunit5AutoExtension.weldInit(WeldJunit5AutoExtension.java:75)
	at org.jboss.weld.junit5.WeldJunit5Extension.getDefaultInitiator(WeldJunit5Extension.java:310)
	at org.jboss.weld.junit5.WeldJunit5Extension.lambda$startWeldContainerIfAppropriate$0(WeldJunit5Extension.java:238)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at org.jboss.weld.junit5.WeldJunit5Extension.startWeldContainerIfAppropriate(WeldJunit5Extension.java:238)
	at org.jboss.weld.junit5.WeldJunit5Extension.beforeEach(WeldJunit5Extension.java:218)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeEachCallbacks$2(TestMethodTestDescriptor.java:166)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeBeforeMethodsOrCallbacksUntilExceptionOccurs$6(TestMethodTestDescriptor.java:202)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeMethodsOrCallbacksUntilExceptionOccurs(TestMethodTestDescriptor.java:202)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeBeforeEachCallbacks(TestMethodTestDescriptor.java:165)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:132)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
```

--- a/jans-auth-server/server/src/test/java/io/jans/as/server/comp/db/UserJansExtUidAttributeTest.java
+++ b/jans-auth-server/server/src/test/java/io/jans/as/server/comp/db/UserJansExtUidAttributeTest.java
@@ -6,28 +6,6 @@
 
 package io.jans.as.server.comp.db;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-
-import java.io.File;
-import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
-
-import org.jboss.weld.junit5.ExplicitParamInjection;
-import org.jboss.weld.junit5.auto.AddBeanClasses;
-import org.jboss.weld.junit5.auto.EnableAutoWeld;
-import org.jboss.weld.junit5.auto.WeldJunit5AutoExtension;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
-import org.junit.jupiter.api.Order;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestMethodOrder;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.slf4j.Logger;
-
 import io.jans.as.common.model.common.User;
 import io.jans.as.common.service.common.ApplicationFactory;
 import io.jans.orm.PersistenceEntryManager;
@@ -44,6 +22,24 @@ import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.enterprise.inject.Default;
 import jakarta.enterprise.inject.Produces;
 import jakarta.inject.Inject;
+import org.jboss.weld.junit5.ExplicitParamInjection;
+import org.jboss.weld.junit5.auto.AddBeanClasses;
+import org.jboss.weld.junit5.auto.EnableAutoWeld;
+import org.jboss.weld.junit5.auto.WeldJunit5AutoExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.slf4j.Logger;
+
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * @author Yuriy Movchan Date: 05/12/2024
@@ -55,8 +51,7 @@ import jakarta.inject.Inject;
 @AddBeanClasses(io.jans.service.util.Resources.class)
 
 @AddBeanClasses(value = { io.jans.orm.service.PersistanceFactoryService.class,
-		io.jans.orm.ldap.impl.LdapEntryManagerFactory.class, io.jans.orm.sql.impl.SqlEntryManagerFactory.class,
-		io.jans.orm.couchbase.impl.CouchbaseEntryManagerFactory.class, io.jans.orm.cloud.spanner.impl.SpannerEntryManagerFactory.class})
+		io.jans.orm.ldap.impl.LdapEntryManagerFactory.class, io.jans.orm.sql.impl.SqlEntryManagerFactory.class})
 
 @ExplicitParamInjection
 public class UserJansExtUidAttributeTest {

--- a/jans-auth-server/server/src/test/java/io/jans/as/server/comp/db/UserServiceTest.java
+++ b/jans-auth-server/server/src/test/java/io/jans/as/server/comp/db/UserServiceTest.java
@@ -6,26 +6,6 @@
 
 package io.jans.as.server.comp.db;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-
-import java.nio.file.Paths;
-import java.util.Arrays;
-import java.util.Properties;
-
-import org.jboss.weld.junit5.ExplicitParamInjection;
-import org.jboss.weld.junit5.auto.AddBeanClasses;
-import org.jboss.weld.junit5.auto.EnableAutoWeld;
-import org.jboss.weld.junit5.auto.WeldJunit5AutoExtension;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
-import org.junit.jupiter.api.Order;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestMethodOrder;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.slf4j.Logger;
-
 import io.jans.as.common.model.common.User;
 import io.jans.as.common.service.common.ApplicationFactory;
 import io.jans.model.user.authenticator.UserAuthenticator;
@@ -44,6 +24,23 @@ import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.enterprise.inject.Default;
 import jakarta.enterprise.inject.Produces;
 import jakarta.inject.Inject;
+import org.jboss.weld.junit5.ExplicitParamInjection;
+import org.jboss.weld.junit5.auto.AddBeanClasses;
+import org.jboss.weld.junit5.auto.EnableAutoWeld;
+import org.jboss.weld.junit5.auto.WeldJunit5AutoExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.slf4j.Logger;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * @author Yuriy Movchan Date: 18/11/2024
@@ -55,8 +52,7 @@ import jakarta.inject.Inject;
 @AddBeanClasses(io.jans.service.util.Resources.class)
 
 @AddBeanClasses(value = { io.jans.orm.service.PersistanceFactoryService.class,
-		io.jans.orm.ldap.impl.LdapEntryManagerFactory.class, io.jans.orm.sql.impl.SqlEntryManagerFactory.class,
-		io.jans.orm.couchbase.impl.CouchbaseEntryManagerFactory.class, io.jans.orm.cloud.spanner.impl.SpannerEntryManagerFactory.class})
+		io.jans.orm.ldap.impl.LdapEntryManagerFactory.class, io.jans.orm.sql.impl.SqlEntryManagerFactory.class})
 
 @ExplicitParamInjection
 public class UserServiceTest {

